src/Bridge/Symfony/Serializer/JobExecutionNormalizer.php

```diff
@@ -147,6 +149,8 @@ private function failureFromArray(array $array): Failure
             $array['class'],
             $array['message'],
             $array['code'],
+            $array['file'],
```

Comment:
Previous executions wont have those entries, and this will result as a failure, maybe adding a default value like `unknown:0`